server/lib/routes/get-metrics-flow.js

```diff
@@ -44,6 +46,14 @@ module.exports = function (config) {
       type: FLOW_EVENT_NAME
     }, metricsData, req);
 
+    if (metricsData.form_type === FORM_TYPE_EMAIL) {
+      logFlowEvent({
+        flowTime: flowBeingTime,
+        time: flowBeingTime,
```

Comment:
I can see this typo was already here, but still maybe worth fixing: `flowBeginTime`

Reply:
Thanks Phil, will fix up!